client/src/services/isTwitterConnected.js

```diff
@@ -0,0 +1,5 @@
+import { store } from '../redux/store';
+
+export const isTwitterConnected = () => {
```

Comment:
Why create a service to ask for twitter connection if you already have it in the reducer?

Reply:
This checks if info about connected user was set in the redux store. 
isConnected property is set to true when an account was successfully connected

Reply:
Update
Deleted this as it was redundant